distributed/scheduler.py

```diff
@@ -2054,6 +2065,16 @@ def remove_client(self, client=None):
                                       client=cs.client_key)
             del self.clients[client]
 
+        @gen.coroutine
+        def remove_client_from_events():
+            # If the client isn't registered anymore after the delay, remove from events
+            if client not in self.clients and client in self.events:
+                del self.events[client]
```

Comment:
Style request.  Can I ask for a blank line between the function definition and the following line?